core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -91,45 +156,119 @@ Hudson.ReadPermission.Description=\
   Ce droit est utile quand vous ne voulez pas que les utilisateurs non authentifiés puissent voir les pages Jenkins \
   — retirez ce droit à l''utilisateur anonymous, puis \
   ajoutez le pseudo-utilisateur "authenticated" et accordez-lui le droit en lecture.
+Hudson.RunScriptsPermission.Description=\
+  Déprécié - veuillez utiliser le droit Overall/Administer à la place
+Hudson.NodeDescription=le nœud principal du contrôleur Jenkins
 
 Item.Permissions.Title=Job
-
+Item.CREATE.description=Crée un nouveau job.
+Item.DELETE.description=Supprime un job.
+Item.CONFIGURE.description=Modifie la configuration d''un job.
+Item.READ.description=Lire un job. (Vous pouvez interdire ce droit, mais autoriser Découvrir pour forcer un utilisateur anonyme à s''identifier et voir le job.)
```

```suggestion
Item.READ.description=Lire un job. (Vous pouvez interdire ce droit, mais autoriser "Découvrir" pour forcer un utilisateur anonyme à s''identifier et voir le job.)
```
Easier identification